paper/paper.md

```diff
@@ -54,6 +54,12 @@ contains a feature of setting thresholds for different QC criteria based on inst
 type and evaluates these criteria automatically, which to the best of our knowledge, is not supported
 by existing software.
 
+Another software library which focuses on working with Illumina data from a sequencing run perspective
+is basecallQC [@basecallQC]. This is a library for the R [@R] programming language, which allows the user
+to perform utility actions related to a sequencing run, as well as create summary reports.
+It does not, however, provide built in functionality to access quality control thresholds, nor does it
```

Comment:
Suggestion: "It does not, however, provide built in functionality to assess whether quality control thresholds are fulfilled or not, nor does it [...]"